### PR TITLE
[Method References 2] LPS-176632 Convert method reference to ServiceTrackerMap in class CompositeFacetProcessor(module portal-search-solr8-impl)

### PR DESCRIPTION
**Pull request by W2jx**
resend, Code has been modified as per https://github.com/liferay-core-infra/liferay-portal/pull/4068#discussion_r1161039551.

Previous: https://github.com/liferay-core-infra/liferay-portal/pull/4070

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:search-solr

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by vicnate5** [comment]
No regressions found.

**Comment by W2jx** [comment]
no conflicts(4/11 10:46)

**Comment by W2jx** [comment]
no conflicts(4/12 9:52)

File: modules/apps/portal-search-solr8/portal-search-solr8-impl/src/test/java/com/liferay/portal/search/solr8/internal/search/engine/adapter/search/SearchRequestExecutorFixture.java [hunk 1]
```diff
@@ -313,6 +323,8 @@ protected void setSolrClientManager(SolrClientManager solrClientManager) {
 		_solrClientManager = solrClientManager;
 	}
 
+	private static BaseSolrQueryAssemblerImpl _baseSolrQueryAssemblerImpl;
```
Comment (tinatian) [inline modules/apps/portal-search-solr8/portal-search-solr8-impl/src/test/java/com/liferay/portal/search/solr8/internal/search/engine/adapter/search/SearchRequestExecutorFixture.java:326]: @W2jx If this _baseSolrQueryAssemblerImpl is going to be assigned to different value for each test, then it should not be "static", if it is non-static and you do not need to clean it in tear down either.

**Comment by tinatian** [comment]
@W2jx Please see my comment and check your other similar changes, and resend.

Thanks
Tina

**Pull request closed without merging**